engine/src/main/java/com/datatorrent/stram/client/StramClientUtils.java

```diff
@@ -118,6 +118,8 @@
   public static final long DELEGATION_TOKEN_MAX_LIFETIME_DEFAULT = 7 * 24 * 60 * 60 * 1000;
   public static final String TOKEN_REFRESH_PRINCIPAL = StramUserLogin.DT_AUTH_PREFIX + "token.refresh.principal";
   public static final String TOKEN_REFRESH_KEYTAB = StramUserLogin.DT_AUTH_PREFIX + "token.refresh.keytab";
+  public static final String CUSTOM_SSL_KEYSTORE_CONFIG = StreamingApplication.APEX_PREFIX + "ssl.keystore.config";
```

Comment:
Regarding STRAM_HTTP_CUSTOM_CONFIG this was added in APEXCORE-711 and serves a slightly different purpose. If the SSL keystore files are already present on the app master node (which means they don't need to be copied/deployed to the target app master node) then APEXCORE-711 functionality is sufficient and you just need to use STRAM_HTTP_CUSTOM_CONFIG attribute.

Whereas this JIRA is needed if the SSL keystore is not present on the target app master node so the user needs the Stram client to package it and include it in the app package to be deployed. Let me know if you need more info - you can also read the description in the JIRA